cli/src/main.rs

```diff
@@ -78,8 +94,13 @@ fn main() {
 /// - `start_block` is greater than `end_block`.
 /// - Not enough blocks in the database to cover the required range of blocks to
 ///   replay.
-/// - Any error during execution of `starknet-replay`.
-fn run(start_block: u64, end_block: u64, database_path: PathBuf) -> anyhow::Result<()> {
+/// - Any error during execution of `cairo-replay`.
+fn run(
+    start_block: u64,
+    end_block: u64,
+    database_path: PathBuf,
+    svg_path: Option<PathBuf>,
```

Comment:
This list is likely to get bigger and bigger as the various capabilities are enhanced. I would suggest just passing the `Args` struct here instead, as it does all the validation anyway!